### ISSUE TITLE
Is there any result in this code?

### ISSUE THREAD

**Issue by 462525023** [pippy360/codeToTrainFCRN#1]
Is there any result like accuracy according to the test csv?
Thank you!

**Comment by pippy360** [comment]
No, this code isn't complete and is still in development. 

Check out this repo for results/the official paper and everything:
https://github.com/iro-cp/FCRN-DepthPrediction

**Comment by 462525023** [comment]
@pippy360  
Thanks!
And if it has not completed, is the train step correct?

**Comment by pippy360** [comment]
No, the train step is not yet correct. I am working on it but I'm busy at the moment so it could be months
